### PR TITLE
New tile C8Y IoT Styleguide

### PR DESCRIPTION
**Pull request by MWindel**
Added the C8Y IoT Styleguide as a new tile to the App enablement section

### CODE REVIEW

File: content/c8y-iot-styleguide.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Cumulocity IoT Styleguide
+icon: "dlt-c8y-icon-thin-client"
+type: root
+layout: redirect
+external: ""
+audience:
+  - Developers
+  - Users
```
Comment (BeateRixen) [inline content/c8y-iot-styleguide.md:9]: The style guide is for developers only, I´d say.

File: content/c8y-iot-styleguide.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Cumulocity IoT Styleguide
+icon: "dlt-c8y-icon-thin-client"
```
Comment (BeateRixen) [inline content/c8y-iot-styleguide.md:3]: We should use a different icon here, not the one for the thin-client. I assume that the Style guide has an own icon?

File: content/c8y-iot-styleguide.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  - Developers
+  - Users
+  - Business
+bundle: app_enablement
+section:
+- app_enablement
+weight: 90
+---
+
+Learn more about the Cumulocity IoT UI guidelines to create interfaces consistent with the Cumulocity principles and best practices.
```
Comment (BeateRixen) [inline content/c8y-iot-styleguide.md:17]: consistent > consistently ?

File: content/c8y-styleguide/ui-styleguide.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+weight: 15
```
Comment (BeateRixen) [inline content/c8y-styleguide/ui-styleguide.md:2]: If we use a direct link we do not need a landing page.

File: content/c8y-iot-styleguide.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: Cumulocity IoT Styleguide
+icon: "dlt-c8y-icon-thin-client"
+type: root
+layout: redirect
+external: ""
```
Comment (BeateRixen) [inline content/c8y-iot-styleguide.md:6]: For now, I would add the link to the current UI style guide, as this is the publicly available one. If this changes, we probably must also change the name or at least mention "codex" in the description?

File: content/c8y-iot-styleguide.md (hunk 1 above)
Comment (MWindel) [reply content/c8y-iot-styleguide.md:9]: Done.

File: content/c8y-iot-styleguide.md (hunk 3 above)
Comment (MWindel) [reply content/c8y-iot-styleguide.md:17]: It refers to "to be consistent with".

File: content/c8y-iot-styleguide.md (hunk 5 above)
Comment (MWindel) [reply content/c8y-iot-styleguide.md:6]:
```suggestion
external: "https://styleguide.cumulocity.com/#/"
```

File: content/c8y-styleguide/ui-styleguide.md (hunk 4 above)
Comment (MWindel) [reply content/c8y-styleguide/ui-styleguide.md]: Done.

File: content/c8y-iot-styleguide.md (hunk 2 above)
Comment (MWindel) [reply content/c8y-iot-styleguide.md:3]: I already pinged @carlosceia regarding the icon issue.

**Review by rssag — APPROVED** [review]

**Pull request merged**